src/AppBundle/Service/ReportService.php

```diff
@@ -203,4 +205,44 @@ public function submit(Report $currentReport, User $user, \DateTime $submitDate)
 
         return $newYearReport;
     }
+
+    /**
+     * If one report started, return the other nonStarted reports with the same start/end date
+     *
+     *
+     * @param Collection $reports
+     *
+     * @return Report[] indexed by ID
+     */
+    public function findDeleteableReports(Collection $reports)
+    {
+        $reportIdToStatus = [];
+        foreach($reports as $ur) {
+            $reportIdToStatus[$ur->getId()] = [
+                'status'=> $ur->getStatus()->getStatus(),
+                'start'=> $ur->getStartDate()->format('Y-m-d'),
+                'end' => $ur->getEndDate()->format('Y-m-d'),
+                'sections' => $ur->getStatus()->getSectionStatus()
+            ];
+        }
+
+        $ret = [];
+        foreach($reports as $report1) {
+            foreach($reports as $report2) {
+                if ($report1->getId() === $report2->getId()) {
+                    continue;
+                }
+                // find report with same date that have not started
+                if ($report1->getStatus()->hasStarted()
```

Comment:
Should this be negated? comment says    // find report with same date that have not started